StyleGuide.md

```diff
@@ -55,6 +55,7 @@ if (x == FOO) {
 ### Naming
 
 * typedefs should always end in `_t`.
+* function pointer type defs endin `_fn_t`
```

Comment:
```suggestion
* function pointer type defs end in `_fn_t`
```

Reply:
Darn, I was wondering if they had a way for you to accept the suggestion directly.